### DISCUSSION TITLE
Roku SDK

### DISCUSSION THREAD

**Discussion by johnsonelahi** [rrirower/DP2SceneGraphWizard#86] [Q&A]
Hello, Can you add the attribute to manifest for overlay light background when if select any video. See example
![dev](https://github.com/rrirower/DP2SceneGraphWizard/assets/95589772/3fb19dc6-071d-47c9-8fc4-dd43923156f1)

**Comment by rrirower** [comment]
Have you tried using the manifest optional attribute [USE_THUMBNAIL_BACKGROUND](https://github.com/rrirower/DP2SceneGraphWizard/wiki/Custom-Manifest-Attributes)?

**Comment by rrirower** [comment]
Do you have an update on this?

**Reply by johnsonelahi** [reply]
Hello. How are you. I have sent you a screenshot. The thumbnail of the video is visible on the full screen, but the problem is that the fonts are not visible clearly on the screen. You specify an attribute that dims the screen behind the video thumbnail. Which makes all the fonts like title, description etc will be visible clearly. Also please check the screenshot. I have used your software for channel.
![WhatsApp Image 2024-03-05 at 2 29 14 AM](https://github.com/rrirower/DP2SceneGraphWizard/assets/95589772/4551256e-13e4-4d58-a037-232dde491207)

**Reply by johnsonelahi** [reply]
I want the background screen with thumbnail like attached. 
![308538344-3fb19dc6-071d-47c9-8fc4-dd43923156f1](https://github.com/rrirower/DP2SceneGraphWizard/assets/95589772/f988d5ba-994a-437e-8ddc-4dc576f61290)

**Reply by rrirower** [reply]
You can change the font and color.  Please see the [Wiki](https://github.com/rrirower/DP2SceneGraphWizard/wiki/How-to:-Change-the-text-color-scheme) and discussion #89
